main.go

```diff
@@ -60,6 +60,8 @@ func mainHandler(c *gin.Context) {
 	//	return
 	//}
 
+	c.Set("ifconfig_hostname", getEnvWithDefault("HOSTNAME", "ifconfig.io"))
```

Comment:
Can we move this to a global variable initialized with `init()`?

Looking up environment variables is a syscall. ifconfig.io now gets 3-5k requests per second, so any minimization of syscalls is very valuable.